pages/blog/2020-summary.md

```diff
@@ -0,0 +1,929 @@
+
+Slack hard numbers confirm what we see in Slack every day. Luckily, even though the activity doubled, the number of active users that help out in the workspace also grew :pray:. With the recently announced Postman partnership, I do not worry about how we'll handle 2021.
+
+Next year we need to address two critical topics:
+- out community grows, but the knowledge that we all share in Slack is lost. We use a free version of Slack, and therefore, only the last 10k messages are available. In 2021 we need to make a strong decision on what next. Should we spend some budget on Slack commercial subscriptions, or maybe we should find a better tool.
+- we need to reorganize our channels, be a bit more granular, and target suitable community groups. We'll bother you to get your feedback once we start this reorganization.
+
+## LinkedIn
+
+[LinkedIn](https://www.linkedin.com/company/asyncapi/) grew a lot this year. We won't be able to compare this year with the previous because... this is not a place where I want to discuss not the experience you get with LinkedIn.
+
+### Followers and impressions
+
+At the moment, we have 757 followers.
+
+<table>
+  <tr>
+    <td align="center">
+      Followers
+    </td>
+    <td align="center">
+      Likes
+    </td>
+    <td align="center">
+      Impressions
+    </td>
+  </tr>
+  <tr>
+    <td align="center">
+      611<br/>
+      57 from payed marketing
+    </td>
+    <td align="center">
+      1253
+    </td>
+    <td align="center">
+      80095
+    </td>
+  </tr>
+</table>
+
+People that like your updates are most likely the ones that click on links. 75% (947) of likes converted into visits to our [website](http://asyncapi.com/). We took this data from Google Analytics, and we know that in 2019 we had only 137 visits from LinkedIn.
+
+The most performing update was the one about [partnership with Postman](https://www.linkedin.com/feed/update/urn:li:activity:6742099578591481856). It is LinkedIn, and it is all about business.
+
+The below charts show that data match reality. The highest peak in late March is related to our first [AsyncAPI Online Conference](https://www.youtube.com/watch?v=oMSzGc5bDr4). The rise of sponsored traffic is connected to our post-conference activities using the money we got from conference sponsors.
+
+<Figure
+  src="/img/posts/2020-summary/linkedin-impressions.webp"
+  caption="Figure 2: Impressions to updates in 2020"
+/>
+
+### Sponsored marketing
+
+This year we explored paid adverts on LinkedIn. We had some money left from sponsors of our AsyncAPI conference but zero experience with LinkedIn.
+
+Luckily we have a fantastic community. One of the community members, [Dan Weese](https://www.linkedin.com/in/daniel-weese-53b27410/), helped us here. He did not only put us in contact with marketing expert, [Charles Serdou](https://www.linkedin.com/in/sardou/), but also covered his wages. Thank you Dan :pray:
+
+This way, we covered only LinkedIn costs paying $1 074.08
```

Comment:
```suggestion
This way, we covered only LinkedIn costs paying $1,074.08
```

Reply:
this is actually not how you show money
on transferwise I see `1 074,08` and my bank uses the same format, so I will change everywhere to comma instead of the dot